framework/src/modules/README.md

```diff
@@ -146,11 +159,11 @@ A modules' life cycle consists of following events in the right order:
 
 **Loading**
 
-- _module_:registeredToBus
-- _module_:loading:started
-- _module_:loading:finished
+* _module_:registeredToBus
+* _module_:loading:started
+* _module_:loading:finished
 
 **Unloading**
 
-- _module_:unloading:started
-- _module_:unloading:finished
+* _module_:unloading:started
+* _module_:unloading:finished
```

Comment:
@Tschakki The implementation for these events are not done yet. I had created issue for reference.

Reply:
@nazarhussain Which events exactly?
Can you point me to the issue?

Reply:
nvm I found it ! For reference: #3632